ui/helpers/utils/accounts.js

```diff
@@ -0,0 +1,32 @@
+export function getAccountNameErrorMessage(
+  accounts,
+  context,
+  newAccountName,
+  defaultAccountName,
+) {
+  const isDuplicateAccountName = accounts.some(
+    (item) => item.name === newAccountName,
+  );
+
+  const localizedWordForAccount = context
+    .t('newAccountNumberName')
+    .replace(' $1', '');
+
+  const reservedRegEx = new RegExp(`^${localizedWordForAccount} \\d+$`, 'iu'); // Match strings starting with ${localizedWordForAccount} and then any numeral, case insensitive
+  const isReservedAccountName = reservedRegEx.test(newAccountName);
+
+  const isValidAccountName =
+    newAccountName === defaultAccountName || // What is written in the text field is the same as the placeholder
+    (!isDuplicateAccountName && !isReservedAccountName);
+
+  let errorMessage;
+  if (isValidAccountName) {
+    errorMessage = '\u200d'; // This is Unicode for an invisible character, so the spacing stays constant
```

Comment:
can you explain what you mean about he spacing?

Reply:
If you return `null` or `''`, the dialog grows in height and the buttons move down when an error message appears (shown below).  This is also why I **do** want that div to exist even if there is no `errorMessage`.

![image](https://user-images.githubusercontent.com/539738/207986351-8bebe0af-f5cb-41fa-9d85-b1d97af8846d.png)
![image](https://user-images.githubusercontent.com/539738/207986152-428baff0-cbcc-4cbb-99ac-4a9ccd65bef0.png)